Graphs/TreeGraph/index.js

```diff
@@ -524,12 +524,9 @@ class TreeGraph extends AbstractGraph {
             
             nodeEnter.append('foreignObject')
                 .attr("id", (d) => `node-${d.id}`)
-                .attr('x', rectNode.textMargin)
-                .attr('y', rectNode.textMargin + 12)
-                .attr('width', () => {
-                    return (rectNode.width - rectNode.textMargin * 2) < 0 ? 0 :
-                        (rectNode.width - rectNode.textMargin * 2)
-                })
+                .attr('x', rectNode.textMargin + 10)
```

Comment:
why do you need to change this since rectNode is a configuration? Why not changing that configuration?